charts/managed-serviceaccount/templates/placement.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if not .Values.installByPlacement.placementName }}
```

Comment:
Here I am not sure if the placement and clustersetbinding are needed, maybe it is clearer that the placement is created/controlled by other components. And here we just use the results(which clusters are selected).

@qiujian16 WDYT?

Reply:
yes, I think we do not need to include placement here.